lib/core/directives/node-download.directive.spec.ts

```diff
@@ -26,10 +26,11 @@ import { CoreTestingModule } from '../testing/core.testing.module';
 import { TranslateModule } from '@ngx-translate/core';
 
 @Component({
-    template: '<div [adfNodeDownload]="selection"></div>'
+    template: '<div [adfNodeDownload]="selection" [version]="version"></div>'
```

Comment:
this needs a documentation update

Reply:
https://github.com/Alfresco/alfresco-ng2-components/pull/5785/files#diff-1b2eada0dcb743cd0b35f51264138b3c